package.json

```diff
@@ -14,7 +14,7 @@
     "@dvsa/cookie-manager": "^1.0.*",
     "autoprefixer": "^10.4.19",
     "chosen-npm": "^1.4.2",
-    "govuk-frontend": "4.8.0",
+    "govuk-frontend": "^5.4.1",
```

Comment:
Best to avoid manual changes as the `package-lock.json` doesn't reflect the changes.